.azure-pipelines/ipu-tests.yml

```diff
@@ -68,16 +68,23 @@ jobs:
         set -eux
         source ${{ variables.poplar_sdk }}/poplar-ubuntu*/enable.sh
         source ${{ variables.poplar_sdk }}/popart-ubuntu*/enable.sh
-
         python -c "import poptorch; print(poptorch.__version__)"
       displayName: "Check poptorch installation"
 
     - bash: |
         source ${{ variables.poplar_sdk }}/poplar-ubuntu*/enable.sh
         source ${{ variables.poplar_sdk }}/popart-ubuntu*/enable.sh
-        export POPTORCH_WAIT_FOR_IPU=1
-        export PL_RUN_IPU_TESTS=1
-        python -m coverage run --source pytorch_lightning -m pytest tests -vv --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=50
+        cd src
+        python -m pytest pytorch_lightning
+      displayName: 'DocTests'
```

Comment:
I wouldn't add doctesting to these accelerator jobs as it complicates the pipeline for contributors